transport/src/main/java/io/netty/channel/ChannelConfig.java

```diff
@@ -195,19 +195,19 @@
     ChannelConfig setAutoRead(boolean autoRead);
 
     /**
-     * @deprecated From version 5.0, {@link Channel} will not be closed on write failure.
-     *
      * Returns {@code true} if and only if the {@link Channel} will be closed automatically and immediately on
      * write failure.  The default is {@code false}.
+     *
+     * @deprecated
      */
     @Deprecated
     boolean isAutoClose();
 
     /**
-     * @deprecated From version 5.0, {@link Channel} will not be closed on write failure.
-     *
      * Sets whether the {@link Channel} should be closed automatically and immediately on write faillure.
      * The default is {@code false}.
+     *
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the missing @deprecated Javadoc tag. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)